### ISSUE TITLE
Machine Learning

### ISSUE THREAD

**Issue by hkethi002** [hkethi002/guild-master#2]
### Task
Once we have the data, we will need to create a model using scikit-learn that will classify the images we send it as shapes.

**Comment by momja** [comment]
I've been messing around with the SKLearn library. Not sure what algorithm would be best for finding circles. I tested this with the Pima Indians Diabetes Dataset on UCI, using Naive Bayes. It was 77% so not great. Just thought I'd share this. I have a feeling SVM might work better for the circles. 
`from sklearn import datasets, cluster, metrics, svm
from sklearn.preprocessing import scale
from sklearn.model_selection import train_test_split
from sklearn.naive_bayes import GaussianNB
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

import numpy as np
import urllib
# URL for the Pima Indians Diabetes dataset (UCI Machine Learning Repository)
url = "circles.data"
# download the file
raw_data = urllib.urlopen(url)
# load the CSV file as a numpy matrix
dataset = np.loadtxt(raw_data, delimiter=",")
# separate the data from the target attributes
X = dataset[:,0:7]
y = dataset[:,8]

# Split the 'digits' data into training and test sets
X_train, X_test, Y_train, Y_test = train_test_split(X, y, test_size = 0.25, random_state = 42)
# create instance of Gaussian Naive Bayes Algorithm
gnb = GaussianNB()
# use gnb to fit the data based on its classification
gnb_model = gnb.fit(X_train, Y_train)
# predict values that are not already classified
gnb_y_pred = gnb_model.predict(X_test)

# print general metrics to understand the accuracy of the predictions
print(metrics.classification_report(Y_test, gnb_y_pred))
`
